platform/mellanox/mlnx-platform-api/sonic_platform/modules_mgmt.py

```diff
@@ -458,6 +459,25 @@ def power_on_module(self, port, module_sm_obj, dynamic=False):
                 return STATE_HW_NOT_PRESENT
         return STATE_NOT_POWERED
 
+    def update_frequency(self, port, xcvr_api):
+        # first read the frequency support - if it's 1 then continue, if it's 0 no need to do anything
+        module_fd_freq_support_path = SYSFS_INDEPENDENT_FD_FREQ_SUPPORT.format(port)
+        val_int = utils.read_int_from_file(module_fd_freq_support_path)
+        if 1 == val_int:
+            # read the module maximum supported clock of Management Comm Interface (MCI) from module EEPROM.
+            # from byte 2 bits 3-2:
+            # 00b means module supports up to 400KHz
+            # 01b means module supports up to 1MHz
+            logger.log_info(f"check_module_type reading mci max frequency for port {port}")
```

Comment:
can we use log_debug here?

Reply:
I guess we can, but as far as I understand it will only be visible after your logging PR will be merged.

Reply:
Let's set it as debug, it is not information.